### PR TITLE
PSScriptAnalyser clean and removing deprecated functions.

### PR DESCRIPTION
**Pull request by BrianLPeterson**
Note:  Not fully tested.

### CODE REVIEW

File: ComputeAdmin/AzureStack.ComputeAdmin.psm1 [hunk 1]
```diff
@@ -234,155 +245,208 @@ Function Add-AzSVMImage
...
-    {
+    if (Get-AzsVMImage -publisher $publisher -offer $offer -sku $sku -version $version -location $location -ErrorAction SilentlyContinue) {
         Write-Verbose "VM Image is present in Azure Stack - continuing to remove" -Verbose
         $VMImageExists = $true
     }
-    else
-    {
-        Write-Verbose -Message ('VM Image with publisher "{0}", offer "{1}", sku "{2}" is not present and will not be removed. Marketplace item may still be removed.' -f $publisher,$offer,$sku) -ErrorAction Stop
+    else {
+        Write-Verbose -Message ('VM Image with publisher "{0}", offer "{1}", sku "{2}" is not present and will not be removed. Marketplace item may still be removed.' -f $publisher, $offer, $sku) -ErrorAction Stop
```
Comment (wangshayu) [inline ComputeAdmin/AzureStack.ComputeAdmin.psm1]: better make the variable case consistent to the declaration too. $Publisher, $Offer, $Sku

File: ComputeAdmin/AzureStack.ComputeAdmin.psm1 [hunk 2]
```diff
@@ -469,15 +535,17 @@ function New-AzSServer2016VMImage {
                 }
 
                 Write-Verbose -Message "Making VHD bootable"
-                $null = Invoke-Expression -Command "$VHDDriveLetter`:\Windows\System32\bcdboot.exe $VHDDriveLetter`:\windows /s $VHDDriveLetter`: /f BIOS" -ErrorAction Stop
-            } catch {
+                $null = & "$VHDDriveLetter`:\Windows\System32\bcdboot.exe $VHDDriveLetter`:\windows /s $VHDDriveLetter`: /f BIOS" -ErrorAction Stop
```
Comment (wangshayu) [inline ComputeAdmin/AzureStack.ComputeAdmin.psm1]: I think the syntax here should be & "xxx.exe" arg1 arg2
& "$VHDDriveLetter`:\Windows\System32\bcdboot.exe" "$VHDDriveLetter`:\windows" "/s" "$VHDDriveLetter`:" "/f" "BIOS"

Comment (BrianLPeterson) [inline ComputeAdmin/AzureStack.ComputeAdmin.psm1]: good catch bad merge

File: ComputeAdmin/README.md [hunk 3]
```diff
@@ -72,80 +79,78 @@ The command does the following:
...
 You will need to reference your Azure Stack Administrator environment. To create an administrator environment use the below. The ARM endpoint below is the administrator default for a one-node environment.
 
 ```powershell
-Add-AzsEnvironment -Name "AzureStackAdmin" -ArmEndpoint "https://adminmanagement.local.azurestack.external" 
+Add-AzsEnvironment -Name "AzureStackAdmin" -ArmEndpoint "https://adminmanagement.local.azurestack.external"
 ```
 
 ```powershell
-Remove-AzSVMImage -publisher "Canonical" -offer "UbuntuServer" -sku "14.04.3-LTS" -version "1.0.0"
+Remove-AzsVMImage -publisher "Canonical" -offer "UbuntuServer" -sku "14.04.3-LTS" -version "1.0.0" -tenantID <GUID AADTenant> -EnvironmentName "AzureStackAdmin"
```
Comment (MatthewMcGlynn) [inline ComputeAdmin/README.md]: Clean extra auth parameters

File: ComputeAdmin/README.md [hunk 4]
```diff
@@ -72,80 +79,78 @@ The command does the following:
...
-Add-AzsEnvironment -Name "AzureStackAdmin" -ArmEndpoint "https://adminmanagement.local.azurestack.external" 
+Add-AzsEnvironment -Name "AzureStackAdmin" -ArmEndpoint "https://adminmanagement.local.azurestack.external"
 ```
+
 An example usage is the following:
 
 ```powershell
 $path = "<Path to vm extension zip>"
-Add-AzSVMExtension -publisher "Publisher" -type "Type" -version "1.0.0.0" -extensionLocalPath $path -osType Windows
+Add-AzsVMExtension -publisher "Publisher" -type "Type" -version "1.0.0.0" -extensionLocalPath $path -osType Windows
```
Comment (MatthewMcGlynn) [inline ComputeAdmin/README.md]: Same.

File: ComputeAdmin/AzureStack.ComputeAdmin.psm1 (hunk 1 above)
Comment (BrianLPeterson) [inline ComputeAdmin/AzureStack.ComputeAdmin.psm1]: There are a bunch of these... I am inclined to leave it as mismatched casing is acceptable in powershell.   Capitalization of parameters though can look bad in functions properties.   Lets leave it for now.

**Pull request merged**